CONTRIBUTING.md

```diff
@@ -18,16 +18,23 @@
 #
 -->
 
-# Apache Cassandra and Pull Requests
+# How to Contribute
 
-Apache Cassandra doesn't use GitHub pull requests as part of the development process.
-In fact, this repository is a GitHub mirror of [the official repo](https://gitbox.apache.org/repos/asf/cassandra.git).
+There are many opportunities to contribute code to Apache Cassandra, including documentation updates, test improvements,
+bug fixes, changes to the Java code base, and tooling improvements (both Python and Java). The basic workflow is the
+same for all:
+1. Find or create an issue in the [Cassandra JIRA](https://issues.apache.org/jira/browse/CASSANDRA/) that describes the work you plan to do.
+2. Clone the [Apache Cassandra GitHub repo](https://github.com/apache/cassandra).
+3. Create your feature branch. Suggestion: name your branch with the related JIRA id: e.g. CASSANDRA-12345 .
+4. Make, build, test and self-review your changes on your feature branch.
+5. Submit the patch, either by attaching a patch file to your JIRA, creating a GitHub pull request, or posting a
```

Comment:
it seems that ”attaching a patch file to your JIRA“ has not been used for a long time. @michaelsembwever  am i right ?